lib/Word_Lib/Aligned.thy

```diff
@@ -2,7 +2,6 @@
  * Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
  *
  * SPDX-License-Identifier: BSD-2-Clause
-Proofs tidied by LCP, 2024-09
```

Comment:
this is added in other files as well, but you only reverted this one, any reason?

Reply:
I think I started with this file and then later noticed that it's everywhere.